apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -69,43 +69,50 @@ const Ratings: React.FC<{ id: number; tutorName: string | null }> = ({
     );
 
   return (
-    <div className="flex flex-col my-8 mx-10 gap-10  justify-center">
-      {isEmpty(ratings) ? <NoTutorRatings tutorName={tutorName} /> : null}
-
-      <div className="flex gap-4 flex-wrap justify-center">
-        {ratings.map((rating) => {
-          if (
-            "userId" in rating &&
-            (rating.feedback || rating.userId === user.id)
-          )
-            return (
-              <TutorRatingCard
-                feedback={rating.feedback}
-                imageUrl={rating.image}
-                rating={rating.value}
-                studentId={rating.userId}
-                studentName={rating.name}
-                tutorName={tutorName}
-                owner={rating.userId === user.id}
-              />
-            );
+    <div className="flex flex-col y-8 py-8 mx-10 gap-10  justify-center">
+      {isEmpty(ratings) ? (
+        <NoTutorRatings tutorName={tutorName} />
+      ) : (
+        <div className="grid gap-4 flex-wrap justify-center grid-cols-[repeat(auto-fill,minmax(256px,1fr))]">
+          {ratings.map((rating) => {
+            if (
+              "userId" in rating &&
+              (rating.feedback || rating.userId === user.id)
+            )
+              return (
+                <TutorRatingCard
+                  feedback={rating.feedback}
+                  imageUrl={rating.image}
```

Comment:
This will not work. `rating.image` is the image name in our server (e.g., `image.png`). The component expected the full image url which (e.g., https://api.staging.litespace.org/static/image.png). The url depends on the enviorment (local vs staging vs prod). You can use the util named `asFullAssetUrl` to prefix the image with the right url.